internal/operands/tekton-tasks/tekton-tasks.go

```diff
@@ -215,7 +225,7 @@ func (t *tektonTasks) Cleanup(request *common.Request) ([]common.CleanupResult,
 func listDeprecatedClusterTasks(request *common.Request) ([]pipeline.ClusterTask, error) {
 	deprecatedClusterTasks := &pipeline.ClusterTaskList{}
 	err := request.Client.List(request.Context, deprecatedClusterTasks, &client.MatchingLabels{
-		common.AppKubernetesManagedByLabel: common.AppKubernetesManagedByValue,
+		common.AppKubernetesManagedByLabel: "tekton-tasks-operator",
```

Comment:
Can you please make a constant from this variable? It will be required in additional functionality during upgrade

Reply:
Yep, done.